packages/expo-dev-menu-interface/android/src/main/java/expo/interfaces/devmenu/ReactHostWrapper.kt

```diff
@@ -76,7 +77,12 @@ class ReactHostWrapper(reactNativeHost: ReactNativeHost, reactHost: ReactHost?)
   val jsExecutorName: String
     get() {
       if (isBridgelessMode) {
-        return if (reactHost.jsEngineResolutionAlgorithm == JSEngineResolutionAlgorithm.JSC) {
+        // Access private field using reflection
+        val jsEngineResolutionAlgorithmField: Field = reactHost::class.java.getDeclaredField("mJSEngineResolutionAlgorithm")
```

Comment:
could you point out where is the `mJSEngineResolutionAlgorithm` code in ReactHost? i tried to search `mJSEngineResolutionAlgorithm` from 0.76-stable it only appears in `ReactInstanceManagerBuilder` but not ReactHost.

Reply:
Sure, mJSEngineResolutionAlgorithm is defined here https://github.com/facebook/react-native/blob/0.76-stable/packages/react-native/ReactAndroid/src/main/java/com/facebook/react/runtime/ReactHostImpl.java#L133

Reply:
looks like the `mJSEngineResolutionAlgorithm` is not being used and always be null.